src/Javascript/Worker.js

```diff
@@ -84,15 +103,20 @@ const main = async (port) => {
   self.service = service
   self.server = server
 
+  peers.forEach(peer => {
+    latestPeerTimeoutIds[peer] = null
```

Comment:
Is there a reason to do two for loops here? Also, just out of curiosity, any reason to set this to `null`?

Reply:
No good reason for two loops, I'll update that. 😀 

Initializing to `null` is mostly about code readability. A bit more explicit about setting the object up with each of the peers, instead of doing it dynamically when needed.